core/scripts/legacy-javascript/run.js

```diff
@@ -242,43 +264,48 @@ function makeRequireCodeForPolyfill(module) {
 }
 
 async function main() {
-  const pluginGroups = [
-    ...plugins.map(plugin => [plugin]),
-    ['@babel/plugin-transform-regenerator', '@babel/transform-async-to-generator'],
-  ];
-  for (const pluginGroup of pluginGroups) {
-    await createVariant({
+  for (const plugin of plugins) {
+    createVariant({
       group: 'only-plugin',
-      name: pluginGroup.join('_'),
+      name: plugin,
       code: mainCode,
       babelrc: {
-        plugins: pluginGroup,
+        plugins: [plugin],
       },
     });
   }
 
-  for (const coreJsVersion of ['2.6.12', '3.40.0']) {
+  await waitForVariants();
+
+  for (const coreJsVersion of ['3.40.0']) {
```

Comment:
You're leaving this as a `for` loop in case we need to support more corejs versions in the future?